Library/Formula/influxdb.rb

```diff
@@ -11,78 +11,148 @@ class Influxdb < Formula
     sha1 "693a7b3ab9a445a9e681048ecb7aba3582821fe8" => :mountain_lion
   end
 
-  depends_on "leveldb"
-  depends_on "rocksdb"
-  depends_on "autoconf" => :build
-  depends_on "protobuf" => :build
-  depends_on "bison" => :build
-  depends_on "flex" => :build
+  devel do
+    url "https://github.com/influxdb/influxdb.git", :tag => "v0.9.0-rc4"
+    version "0.9.0-rc4"
+  end
+
   depends_on "go" => :build
-  depends_on "gawk" => :build
   depends_on :hg => :build
 
+  if build.stable?
+    depends_on "leveldb"
+    depends_on "rocksdb"
+    depends_on "autoconf" => :build
+    depends_on "protobuf" => :build
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+    depends_on "gawk" => :build
+  end
```

Comment:
We did a complete rewrite of the data store and parser (among other things), so they're pure Go now. No more dependencies!